{{cookiecutter.repo_name}}/MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+graft {{cookiecutter.repo_name}}
+global-exclude *.py[cod] __pycache__ *.so
```

Comment:
I believe we will need:
```
include setup.py
include README.md
include LICENSE
include MANIFEST.in

include versioneer.py
```
as well. Though we usually use recursive-include.

Reply:
According to [the docs](https://docs.python.org/3/distutils/sourcedist.html#manifest), `setup.py` and `README`s should be included by default for _source_ distributions. I will do some checks, but we were not using any MANIFEST template before and it worked fine, right?

Reply:
Including versioneer is the real kicker. There were certain circumstances where the internal `_version.py` was not correctly autogenerated if the a MANIFEST was not present (rare circumstance when triggering CI deployment builds).

The rest I sometimes see, sometimes not. Seems safer to include no?

Reply:
I have checked Django, click, numpy, and others and none of them included `setup.py`. `README` is normally excluded, but [numpy does include it](https://github.com/numpy/numpy/blob/master/MANIFEST.in#L10). 

`graft` is simpler than `recursive-include` if we use `global-exclude`, according to [this post](https://www.reddit.com/r/Python/comments/40s8qw/simplify_your_manifestin_commands/).

... I love the Python packaging universe -.-

Reply:
Im a bit hesitant to `graft` here as it can be very aggressive and grab things people don't think about. But I think its better overall than many `recursive-include`s